test/integration/visitor_can_add_item_to_cart_and_view_cart_test.rb

```diff
@@ -7,15 +7,25 @@ class VisitorCanAddItemToCartAndViewCartTest < ActionDispatch::IntegrationTest
                 price: 1000)
 
     visit items_path
-    assert page.has_content?('Cart: 0')
+
+    within('.cart-count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.